app.py

```diff
@@ -1,57 +1,159 @@
+"""
+This example uses docopt with the built in cmd module to demonstrate an
```

Comment:
I suggest that you edit this docstring. I think saying `this is an example...` in this docstring is confusing - unless you mean for it to be an actual example.